README.md

```diff
@@ -1,2 +1,255 @@
+
+-   [📖 About the Project](#about-project)
+    -   [🛠 Built With](#built-with)
+        -   [Tech Stack](#tech-stack)
+        -   [Key Features](#key-features)
+    -   [🚀 Live Demo](#live-demo)
+-   [💻 Getting Started](#getting-started)
+    -   [Setup](#setup)
+    -   [Prerequisites](#prerequisites)
+    <!-- -   [Install](#install)
+    -   [Usage](#usage)
+    -   [Run tests](#run-tests)
+    -   [Deployment](#deployment) -->
+-   [👥 Authors](#authors)
+-   [🔭 Future Features](#future-features)
+-   [🤝 Contributing](#contributing)
+-   [⭐️ Show your support](#support)
+<!-- -   [🙏 Acknowledgements](#acknowledgements)
+-   [❓ FAQ (OPTIONAL)](#faq) -->
+-   [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 My Personal Portfolio <a name="about-project"></a>
+
+> This is my personal portfolio project in which you will see my best projects, job history and social networks links.
+
+**My Personal Portfolio** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> In this project i use the following technology stack
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">Html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/default.asp">css</a></li>
+  </ul>
+</details>
+
+<!-- <details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details> -->
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> There are three key my porfolio.
+
+-   **You can see my project**
+-   **You can see my work history**
+-   **You can contact me through fill the the form**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+> We will add live demo Demo in future releases. -->
+
+<!-- - [Live Demo Link](https://google.com) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+<!-- > Describe how a new developer could make use of your project. -->
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+To have visual stodio code
+
+### Setup
+
+Clone this repository to your desired folder:
+
+First clone this repository <a href="#https://github.com/sheezu900/My_Potfolio">back to top</a>
+
+<!-- ### Install
+
+Install this project with:
+ -->
+
+<!-- ### Usage
+
+To run the project, execute the following command: -->
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+<!-- ### Run tests
+
+To run tests, run the following command: -->
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+<!-- ### Deployment
+
+You can deploy this project using: -->
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
```

Comment:
As mentioned above, kindly remove this comment block to make your project look more professional and to follow best practices. 😄